test/test.butt

```diff
@@ -37,7 +37,7 @@ start "butt -h show help"
   assert_startwith "${outlines[0]}" "Usage: butt ["
 end
 #######################################
-start "butt with skip=1 option skip test"
+start "skip one test"
```

Comment:
IMO it is better to mention usage in the comment, e.g. `--skip=1 skips the first test`